SeismicMesh/generation/mesh_generator.py

```diff
@@ -395,6 +395,8 @@ def print_msg2(msg):
     DT = _select_cgal_dim(dim)
 
     pfix, nfix = _unpack_pfix(dim, opts, comm)
+    if comm.rank == 0:
+        print_msg1("Constraining " + str(nfix) + " fixed points..")
```

Comment:
f-strings are a nice thing, so if you ever decide to go Python 3.6+ only (which would be reasonable to do now, 3.5 has reached EOL), you could write
```python
f"Constraining {nfix} fixed points..."
```

Reply:
yea, I think I'll do that. I was catering to someone who would only use Python 2.7 but they're leaving the project :]

Reply:
Try
```
find -name "*.py" | xargs pyupgrade --py36-plus
```
then.